Libraries/Microsoft.Teams.Api/ChannelData.cs

```diff
@@ -80,6 +81,34 @@ public class ChannelData
     [JsonPropertyOrder(10)]
     public int? StreamSequence { get; set; }
 
+    /// <summary>
+    /// Information about the users on behalf of whom the action is performed
+    /// </summary>
+    [JsonPropertyName("onBehalfOf")]
+    [JsonPropertyOrder(11)]
+    public IList<OnBehalfOf>? OnBehalfOf { get; set; }
+
+    /// <summary>
+    /// List of teams that a channel was shared with
+    /// </summary>
+    [JsonPropertyName("sharedWithTeams")]
+    [JsonPropertyOrder(12)]
+    public IList<Team>? SharedWithTeams { get; set; }
+
+    /// <summary>
+    /// List of temas that a channel was unshared from
+    /// </summary>
+    [JsonPropertyName("unsharedFromTeams")]
+    [JsonPropertyOrder(13)]
+    public IList<Team>? UnSharedFromTeams { get; set; }
```

Comment:
Hi @aacebo , just a nit that on bot side we don't capitalize the "S". Not sure if that would cause an issue? We have `UnsharedFromTeams`

Reply:
won't make a difference since it will serialize and deserialize to `unsharedFromTeams`